modules/email-subscribe/email-subscribe.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/*
+ * WARNING: This file is distributed verbatim in Jetpack.
+ * There should be nothing WordPress.com specific in this file.
+ *
+ * @hide-in-jetpack
+ */
+class Email_Subscribe {
+
+	static $shortcode = 'jetpack-email-subscribe';
+
+	static $css_classname_prefix = 'jetpack-email-subscribe';
+
+	// Classic singleton pattern:
+	private static $instance;
+	private function __construct() {}
+	static function getInstance() {
```

Comment:
Since the shortcode relies on the Subscriptions module, I would recommend that you check that Jetpack is active and that that the Subscriptions module is active before you do anything.

Reply:
It does not depend on subscription module anymore - we yanked the connection with subscriptions because of GDPR concerns. Now its a standalone thing.